tests/testsuite/TrafficLight_Basic/Makefile

```diff
@@ -44,7 +44,7 @@ test-c:
 
 
 test-llvm:
-	$(OPENGEODE) controller.pr system_strucure.pr --llvm -O$(O)
+	$(OPENGEODE) controller.pr system_structure.pr --llvm -O$(O)
```

Comment:
??

Reply:
This would indeed break the tests, but the LLVM backend is not used anymore.